mender-convert-package

```diff
@@ -212,7 +211,7 @@ disk_create_file_system_from_folder "work/rootfs/" "work/rootfs.img" \
 log_info "Copying root filesystem image to deploy directory"
 run_and_log_cmd "cp --sparse=always work/rootfs.img deploy/${image_name}.${image_fs_type}"
 
-mender_create_artifact "${device_type}" "${artifact_name}" "${image_name}"
+mender_create_artifact "${device_type}" "${MENDER_ARTIFACT_NAME}" "${image_name}"
```

Comment:
Question:
Wrong previously?

Reply:
Not really wrong. It was somewhat safer, as `mender-convert-package` did not rely on what `mender-convert-modify` and just read from fs, but not we had to introduce this dependency as the file might have been removed by user hook in `mender-convert-modify`